src/capi/azext_capi/_help.py

```diff
@@ -32,6 +32,10 @@
   - name: --ephemeral-disks -e
     type: string
     short-summary: Use ephemeral disks
+  - name: --bootstrap-commands
+    type: string
+    long-summary: A YAML file with user-defined pre- and post-VM bootstrap commands
```

Comment:
Can we put a short example of the YAML file format here in the `long-summary`? If it doesn't fit well here, maybe it's time to start some Markdown docs.